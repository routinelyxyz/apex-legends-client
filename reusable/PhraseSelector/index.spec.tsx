import React from 'react';
import {
  render
} from '@testing-library/react';
import { PhraseSelector } from '.';

test('Selects phrase correctly', () => {
  const { container } = render(
    <div>
      <PhraseSelector
        value="Interstellar"
        phrase="inter"
      />
    </div>
  );
  expect(container.firstChild).toMatchSnapshot();
});