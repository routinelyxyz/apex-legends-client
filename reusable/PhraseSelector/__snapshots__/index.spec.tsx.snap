// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selects phrase correctly 1`] = `
<div>
  
  <strong
    class="selection"
  >
    Inter
  </strong>
  stellar
</div>
`;
